src/loaders/svg/tvgSvgUtil.cpp

```diff
@@ -24,6 +24,7 @@
 #include <memory.h>
 #include <ctype.h>
 #include <errno.h>
+#include <stdint.h>
```

Comment:
jfyi, when you introduce a new header, please compare the symbol size (before/after). If it does, please consider not to use if its possible. Many headers generate codes even just including.